proto/cosmos/accounts/v1/query.proto

```diff
@@ -23,3 +27,40 @@ message AccountQueryResponse {
   // response defines the query response of the account.
   bytes response = 1;
 }
+
+// SchemaResponse is the response type for the Query/Schema RPC method.
+message SchemaRequest {
```

Comment:
can we pull this out into its own pr to reduce the total line changed in this pr?

Reply:
@tac0turtle if I do then I cannot implement CLI and test the correct wiring. :(

Reply:
you should be able to since we would merge the other pr first, rebase this and everything would stay the same. just a minor thing to help with review but we can skip for this one

Reply:
I plan to refactor this part; I will make it one query. Fine to do in subsequent PR?